.sauce/config.yml

```diff
@@ -2,28 +2,27 @@ version: 2.1
 apiVersion: v1alpha
 kind: espresso
 sauce:
-  region: us-west-1
+  region: eu-central-1
   # Controls how many suites are executed at the same time (sauce test env only).
   concurrency: 10
   metadata:
     tags:
-      - e2e
-    build: Sample App
+      - junit
+      - espresso
+      - accessibility
+    build: Release $CI_COMMIT_SHORT_SHA
 espresso:
   app: ./app/build/outputs/apk/debug/app-debug.apk
   testApp: ./app/build/outputs/apk/androidTest/debug/app-debug-androidTest.apk
 suites:
-  - name: "Android DevTools Sample App"
+  - name: "Deque Android AxeDevTools Sample App"
     testOptions:
-      clearPackageData: true
-      useTestOrchestrator: true
+      clearPackageData: false
+      useTestOrchestrator: false
```

Comment:
I would recommend keeping clearPackageData to avoid stale build issues.

Reply:
will revert